yarn-project/world-state/src/world-state-db/merkle_tree_db.ts

```diff
@@ -185,7 +185,7 @@ export interface MerkleTreeOperations {
    * This includes all of the current roots of all of the data trees and the current blocks global vars.
```

Comment:
Stale TSDoc ^

Reply:
The statelness being that it is referring to blocks tree?